engine/execution/ingestion/engine.go

```diff
@@ -424,6 +429,19 @@ func (e *Engine) BlockProcessable(b *flow.Header) {
 		return
 	}
 
+	// skips blocks at or above requested stop height
+	stopping := e.stopAtHeight.Try(func(stopHeight uint64, stopCrash bool) bool {
+		if b.Height >= stopHeight {
+			e.log.Warn().Msgf("Skipping execution of %s at height %d because stop has been requested at height %d", b.ID(), b.Height, stopHeight)
+			return true
+		}
```

Comment:
I think this should be `>` instead of `>=` so while we wait for the finalization signal to stop the node, we could executed forks

Reply:
Hmm, stop height is the height new version should take over, so we don't want to executed the block at this height. 
Might be a bit counterintuitive now, but in next step we plan to have a contract with versions table, in a 

200 v0.23
100 v0.22

which makes sense to denote first block to use a version